gravity-forms/gw-set-list-field-rows-by-field-value.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * Gravity Wiz // Gravity Forms // Set Number of List Field Rows by Field Value
+ *
+ * Add/remove list field rows automatically based on the value entered in the specified field. Removes the add/remove
+ * that normally buttons next to List field rows.
+ *
+ * Plugin Name:  Gravity Wiz // Gravity Forms // Set Number of List Field Rows by Field Value
```

Comment:
Let's update plugin name to "Gravity Forms — Set Number of List Field Rows by Field Value".